data/united-states/illinois.csv

```diff
@@ -49,6 +52,7 @@ acdc94d7-84a9-49d4-b604-cdb737ca2834,Cary Ale House,brewpub,208 W Main St Lowr 1
 99bf7864-8bb8-44cb-a933-49a45b5e47af,Corridor Brewery & Provisions,brewpub,3446 N Southport Ave,,,Chicago,Illinois,60657-1420,United States,7732704272,,-87.6643524,41.9447594
 1d2b23a3-7fdf-4f47-b96d-36bd027fe822,Cruz Blanca Brewery,brewpub,904 W Randolph St,,,Chicago,Illinois,60607-2208,United States,3127331975,http://www.cruzblanca.com,-87.6501403,41.8846013
 e8cad295-e28e-4bb1-8101-5de1049846b2,Crystal Lake Brewing Co.,micro,150 N Main St,,,Crystal Lake,Illinois,60014-4433,United States,,,-88.316204,42.2413125
+,D and G Brewing,micro,303 N 4th St Suite A,,,Saint Charles,Illinois,60174-4591,United States,7732032325,-88.31893980690035,41.916144793779665

```

Comment:
I think this should be this instead:

`,D and G Brewing,micro,303 N 4th St Suite A,,,Saint Charles,Illinois,60174-4591,United States,7732032325,,-88.31893980690035,41.916144793779665`

Reply:
Whoops! Thank you! I forgot to put in the website, but just added it in.

Reply:
Awesome! Thank you @buzzamus ! 🍻